interfaces/cython/cantera/utils.pyx

```diff
@@ -31,6 +31,10 @@ def add_directory(directory):
     """ Add a directory to search for Cantera data files. """
     CxxAddDirectory(stringify(directory))
 
+def get_data_directories(sep=':'):
```

Comment:
It shouldn't matter, since its just a string, and we're returning a list, right?